src/mgmt/rpc.h

```diff
@@ -128,6 +131,16 @@ CRT_RPC_DECLARE(mgmt_params_set, DAOS_ISEQ_MGMT_PARAMS_SET,
 CRT_RPC_DECLARE(mgmt_profile, DAOS_ISEQ_MGMT_PROFILE,
 		DAOS_OSEQ_MGMT_PROFILE)
 
+#define DAOS_ISEQ_MGMT_POOL_GET_SVCRANKS /* input fields */	 \
+	((uuid_t)		(gsr_puuid)		CRT_VAR)
+
+#define DAOS_OSEQ_MGMT_POOL_GET_SVCRANKS /* output fields */	 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
false positive when adding new cart RPCs